src/users/users.controller.swagger.desc.ts

```diff
@@ -0,0 +1,23 @@
+export const SWAGGER_DESC_optionsrequest = `
+    List of permitted communication options
+`;
+
+export const SWAGGER_DESC_finduserbyemail = `
+    Returns user by email
+`;
+
```

Comment:
I'm not sure why they used so strange notation for constants) Let's rename all of them using upper snake case

```ts
```suggestion
export const SWAGGER_DESC_OPTIONS_REQ = `
    List of permitted communication options
`;

export const SWAGGER_DESC_FIND_USER_BY_EMAIL = `
    Returns user by email
`;

```